### ISSUE TITLE
variable, attributes_template does not work

### ISSUE THREAD

**Issue by gribouk** [rogro82/hass-variables#53]
Dear All,

I am trying to use variable objects for my project, but the attributes_template do not work for some reason. I made a small example which yelds no result and I fail to understand why:

```
    detectors_list:
        value: 0
        attributes:
            list: "binary_sensor.dining1"

this I call from developer's tools, services, variable.set_variable with data below

variable: detectors_list
value_template: 2
attributes: >  
  { 
    "list": "{{ variable.attributes.list }}" 
  }
```

No metter what I try, the attributes do not change. Would anyone give me a hint why?

The error message from the log:

Logger: homeassistant
Source: custom_components/variable/__init__.py:219
First occurred: 23:57:58 (1 occurrences)
Last logged: 23:57:58

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/variable/__init__.py", line 219, in async_set_variable
    attributes = json.loads(
  File "/usr/local/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not Wrapper

**Comment by Mariusthvdb** [comment]
Check https://github.com/rogro82/hass-variables/issues/47 for the answer

**Comment by gribouk** [comment]
I do not get it.
What I did:

1. Downloaded repository from https://github.com/Wibias/hass-variables
2. I copied new files from the derectory 'variable' to same derectory at HA server.
3. I try new syntax as follows:
```
variable: detectors_list
attributes: 
    list: {{states('variable.detectors_list')}}
```
and nothing happens. no error, but no resul as well:

<template TemplateState(<state variable.detectors_list=0; list=[object Object]=None @ 2020-12-28T01:04:59.898691+02:00>)>no metter what I try to template, it return list = [object Object]...

What did I do wrong?

**Comment by Wibias** [comment]
Also in your previous comment on #47 (Which you deleted) you even mentioned my repository at https://github.com/Wibias/hass-variables . In the README.MD over there is the right & updated format that you need to use.

**Comment by Mariusthvdb** [comment]
@wibias, was that a post for me? don't recall deleting anything tbh.

**Comment by gribouk** [comment]
Yes, there was one. But the context has chaned and I had to change the message.... 

As I discovered, the integration does work if dowloaded from here: https://github.com/Wibias/hass-variables, but DO NOT TEST IT FROM /developer-tools/service. I have no explanation for that, but if you test it from there, no metter what data you feed with your request - it does not template the attributes, explicit values only. When tested from the actuall script/trigger - everything works fine. It took me 3-4 hours of life to discover that. Maybe, in future they correct this, but as of tooday, 28.12.21, it stays like this...
